sys/postpipe.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Ninep Authors.  All rights reserved.
```

Comment:
I don't think we want this in /sys/postpipe.go

Reply:
moved do sys/src/cmd/postpipe.go